web/packages/api/src/toEthereumSnowbridgeV2.ts

```diff
@@ -185,3 +195,295 @@ export const getSnowbridgeDeliveryFee = async (assetHub: ApiPromise, defaultFee?
     }
     return snowbridgeDeliveryFeeDOT
 }
+
+export type DeliveryXcm = {
+    localXcm: any
+    forwardedXcmToBH: any
+    forwardXcmToAH?: any
+    returnToSenderXcm?: any
```

Comment:
@alistair-singh Since Snowbridge V2 has `AliasOrigin` enabled by default, and we introduced an optional `assetClaimer` in this PR, it seems unessessary to build the `returnToSenderXcm` to send assets back to source chain when they get trapped on AH.

I’d prefer to remove this flow for some cleanup, which could also save users some fees.

WDYT?